channel_scanner_interface_name: wpanda0
signal_scanner_interface_name: w2panda0
bt_mac_address: 00:1A:7D:DA:71:15